platform/mellanox/mlnx-platform-api/sonic_platform/component.py

```diff
@@ -132,17 +248,86 @@ def get_firmware_version(self):
         Returns:
             A string containing the firmware version of the component
         """
-        cpld_version_file_list = glob(CPLD_VERSION_FILE_PATTERN)
+        cpld_version_file_list = glob(self.CPLD_VERSION_FILE_PATTERN)
         cpld_version = ''
         if cpld_version_file_list is not None and cpld_version_file_list:
             cpld_version_file_list.sort()
             for version_file in cpld_version_file_list:
-                version = self._read_generic_file(version_file, CPLD_VERSION_MAX_LENGTH)
+                version = self._read_generic_file(version_file, self.CPLD_VERSION_MAX_LENGTH)
                 if not cpld_version == '':
```

Comment:
@stephenxs According to PEP8:
```
For sequences, (strings, lists, tuples), use the fact that empty sequences are false.
```
`if not some_string:` or `if some_string:`

Reply:
fixed